src/regexp.c

```diff
@@ -973,13 +973,14 @@ ungetchr(void)
 /*
  * Get and return the value of the hex string at the current position.
  * Return -1 if there is no valid hex number.
+ * Returns at max INT_MAX
```

Comment:
```suggestion
 * Returns at maximum INT_MAX.
```

Reply:
actually, I think the comment should be removed again. I initially added it when trying to limit the return value to MAX_INT, but I removed that again, since it wasn't working as I wanted it to.